src/main/java/edu/harvard/iq/dataverse/api/Dataverses.java

```diff
@@ -342,15 +343,35 @@ public Response setMetadataRoot( @PathParam("identifier")String dvIdtf, String b
     
     @GET
     @Path("{identifier}/facets/")
+    /**
+     * return list of facets for the dataverse with alias `dvIdtf`
+     */
     public Response listFacets( @PathParam("identifier") String dvIdtf ) {
```

Comment:
You're off the hook on REST Assured tests until I document how to set up a dev environment to run them, which I'd like to to as part of #3431. The short of it is that you have to run `setup-all.sh --insecure` like in https://github.com/IQSS/dataverse/blob/v4.6/scripts/deploy/phoenix.dataverse.org/post#L3